lib/Sema/TypeCheckStorage.cpp

```diff
@@ -1447,26 +1471,47 @@ synthesizeObservedSetterBody(AccessorDecl *Set, TargetImpl target,
     SetterBody.push_back(Call);
   };
 
+  // Walk the didSet to check if we have a reference to the oldValue
+  auto foundOldValueRef = [&](const AccessorDecl *decl) {
+    bool foundOldValue = false;
+
+    for (auto element : decl->getBody()->getElements()) {
+      if (auto expr = element.dyn_cast<Expr *>()) {
+        expr->forEachChildExpr([&](Expr *subExpr) -> Expr * {
+          if (auto DRE = dyn_cast<DeclRefExpr>(subExpr)) {
+            if (DRE->getDecl()->getBaseName().getIdentifier().is("oldValue")) {
```

Comment:
Would be better to use ctx.Id_oldValue

Reply:
It didn't exist, but I have added one now.